diff -Nur -x '*.orig' -x '*~' sip-tester-3.1//call.cpp sip-tester-3.1.new//call.cpp
--- sip-tester-3.1//call.cpp	2008-04-10 00:50:37.000000000 +1000
+++ sip-tester-3.1.new//call.cpp	2010-06-13 15:30:21.000000000 +1000
@@ -41,6 +41,7 @@
 #include <iostream>
 #include <sys/types.h>
 #include <sys/wait.h>
+#include <limits.h>
 
 #ifdef PCAPPLAY
 #include "send_packets.h"
diff -Nur -x '*.orig' -x '*~' sip-tester-3.1//scenario.cpp sip-tester-3.1.new//scenario.cpp
--- sip-tester-3.1//scenario.cpp	2008-03-29 07:13:47.000000000 +1100
+++ sip-tester-3.1.new//scenario.cpp	2010-06-13 15:30:43.000000000 +1000
@@ -27,6 +27,7 @@
  */
 
 #include <stdlib.h>
+#include <limits.h>
 #include "sipp.hpp"
 #ifdef HAVE_GSL
 #include <gsl/gsl_rng.h>
